consensus/hotstuff/cruisectl/block_time_controller_test.go

```diff
@@ -232,50 +232,78 @@ func (bs *BlockTimeControllerSuite) TestInit_EpochFallbackTriggered() {
 	bs.AssertCorrectInitialization()
```

Comment:
I think we can remove this test case (`TestInit_EpochFallbackTriggered`) because the component shouldn't need to behave differently when EFM is triggered. 

Ideally we would replace this test with one where we initialize the component when the current epoch already contains extensions. However, I'm not sure that we can usefully test that case at this level because we are mocking out the `protocol.Epoch`.

Reply:
https://github.com/onflow/flow-go/pull/6102/commits/a221aee99638fd22c68156f45e2ceff6e9f734ed , are there any tests that don't mock the `protocol.Epoch`?